bucket/git-credential-manager-core.json

```diff
@@ -0,0 +1,22 @@
+{
+    "version": "2.0.475",
+    "description": "Git Credential Manager Core (GCM Core) is a secure Git credential helper built on .NET that runs on Windows and macOS. Linux support is in an early preview.",
+    "homepage": "https://github.com/microsoft/Git-Credential-Manager-Core",
+    "license": "MIT",
+    "url": "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v2.0.475/gcmcore-win-x86-2.0.475.64295.zip",
+    "hash": "498069454e7759364dd1c168a25d1f593d9295db521d4ec33e5897ce1344ea68",
+    "bin": "git-credential-manager-core.exe",
+    "shortcuts": [
+        [
+            "git-credential-manager-core.exe",
+            "GCM"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/microsoft/Git-Credential-Manager-Core",
+        "regex": "/releases/tag/v([\\d.]+)"
+    },
+    "autoupdate": {
+        "url": "https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v$version/gcmcore-win-x86-$version.$matchRandom.zip"
```

Comment:
Cannot work

Reply:
@Ash258  @r15ch13  ok 

the URL has a 5 digit suffix at the end - can you tell me how to match that?

e.g. 41365 in the following url
https://github.com/microsoft/Git-Credential-Manager-Core/releases/download/v2.0.474/gcmcore-win-x86-2.0.474.41365.zip

Reply:
btw $matchRandom is from examples of existing apps in the repo.

Reply:
you use a "$match**_Random_**" but this means the "**Random**" match has to be done in the checkver:regex, which you don't have.